curriculum/js2/3.js

```diff
@@ -6,8 +6,23 @@
  * @returns {array}
  */
 
-const solution = (a) => {
-  return [];
+const isPrime = (num) => {   //return if num isPrime              
```

Comment:
1. you should be doing this recursively. 

2. Import your previous function so you don't have duplicate code.